onepager.py

```diff
@@ -188,10 +196,10 @@
 
 nums = [1, 2, 3, 4]
 for num in nums:
-    print(num)
+    print(num)                                  # prints 1-4, one num per line
 
 for i in range(10):
-    print(i)
+    print(i)                                    # prints 0-9, one num per line. Note num after range is exclusive
```

Comment:
**suggestion**: maybe just "Note range end argument is exclusive." Really though `[start, end)` with an option to pass in the start `range(0, 10)` would be identical to `range(10)`. Showing that might be good.

Reply:
Okay, see changes.